### ISSUE TITLE
ISCLS 2024

### ISSUE THREAD

**Issue by arnabbht** [acl-org/acl-anthology#3071]
### General information about this request

- [X] I confirm that I have read the [Information for Submitters](https://aclanthology.org/info/contrib/).
- [ ] I am submitting a request for a **new venue** that does not exist in the ACL Anthology yet.

### Venue Identifier

iscls

### Volume Title

Proceedings of the 7th International Sanskrit Computational Linguistics Symposium

### Date of Publication

2024-02-14

### Supporting Information

Website: https://iscls.github.io/
Announcement on ACL page: https://www.aclweb.org/portal/content/7th-international-sanskrit-computational-linguistics-symposium-7th-iscls

**Comment by anthology-assist** [comment]
@arnabbht Thank you for the issue. Please submit the ingestion material asap.

**Comment by arnabbht** [comment]
Whom should I email the link to the proceedings file?

Regards,
Arnab


--
Arnab Bhattacharya
অর্ণব ভট্টাচার্য
Professor,
Dept. of Computer Science and Engineering,
Indian Institute of Technology, Kanpur.

**Comment by arnabbht** [comment]
I have sent the data tgz to ***@***.***

I hope that is enough.

We could not put the ISBN number in the proceedings front matter.
So, if we can get that, we will put it.

Thank you.
Regards,
Arnab


--
Arnab Bhattacharya
অর্ণব ভট্টাচার্য
Professor,
Dept. of Computer Science and Engineering,
Indian Institute of Technology, Kanpur.

**Comment by arnabbht** [comment]
There was an error in one of the author fields.
I have updated the tgz file.

Please use this updated file.

The file link is: 
https://drive.google.com/file/d/1H3g5Cv_4jXw7monm0QX2bJL2dpmVddhL/view?usp=drive_link

Thank you.
Regards,
Arnab


--
Arnab Bhattacharya
অর্ণব ভট্টাচার্য
Professor,
Dept. of Computer Science and Engineering,
Indian Institute of Technology, Kanpur.

**Comment by anthology-assist** [comment]
@arnabbht Requested access.

**Comment by arnabbht** [comment]
I had given access.
Please verify.

**Comment by anthology-assist** [comment]
@arnabbht The `2024.iscls.0.bib` is missing in the ingestion material. Could you upload?

**Comment by arnabbht** [comment]
It has been uploaded now.

Thank you.
Regards,
Arnab


--
Arnab Bhattacharya
অর্ণব ভট্টাচার্য
Professor,
Dept. of Computer Science and Engineering,
Indian Institute of Technology, Kanpur.

**Comment by anthology-assist** [comment]
@arnabbht I still don't see the `2024.iscls.0.bib` in the `bib` dir. Could you double check?
![Screenshot 2024-03-08 at 8 21 24 AM](https://github.com/acl-org/acl-anthology/assets/126604033/9d134c24-b167-495c-8b9b-c0797b122f73)

**Comment by hrishikeshrt** [comment]
@anthology-assist Can you please check now.
The bib file `2024.iscls.0.bib` should now exist with the following content.

```
@proceedings{iscls-24,
	title = {Proceedings of the 7th International Sanskrit Computational Linguistics Symposium},
	editor = {Arnab Bhattacharya},
	month = feb,	
	year = {2024},
	address = {Auroville, Puducherry, India},
	publisher = {Association for Computational Linguistics},
	url = {}
}```

The URL field is empty here as I do not know the final URL to the proceedings

**Comment by arnabbht** [comment]
Thank you!Regards,Arnab--Arnab Bhattacharyaঅর্ণব ভট্টাচার্যProfessor,Dept. of Computer Science and Engineering,Indian Institute of Technology Kanpur
-------- মূল বার্তা --------থেকে: Matt Post ***@***.***> তারিখ: 23/3/24  6:34 PM  (GMT+05:30) প্রতি: acl-org/acl-anthology ***@***.***> Cc: Arnab Bhattacharya ***@***.***>, Mention ***@***.***> বিষয়: Re: [acl-org/acl-anthology] ISCLS 2024 (Issue #3071) 
Closed #3071 as completed via 650e75e.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/acl-org/acl-anthology/issues/3071#event-12224381537",
"url": "https://github.com/acl-org/acl-anthology/issues/3071#event-12224381537",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
